src/tableau-area/tableau-area.html

```diff
@@ -14,13 +14,26 @@
 
       .stacks-row {
         @apply --layout-flex;
+        width:1200px;
+      }
+      .stacks-row .tableau-top{
+        vertical-align:top;
+
+      }
+      .stacks-row .tableau-bottom{
+      vertical-align:bottom:
       }
 
     </style>
 
     <div class="stacks-row">
-      <template is="dom-repeat" items="[[stacks]]">
-        <card-stack stack="[[item]]"></card-stack>
+      <template is="dom-repeat" items="[[stacks]]" index-as="stack_no">
```

Comment:
`dom-repeat` always makes the index available as a local variable called `index` so you can just use that. I don't think you need to rename it here unless you were doing it for readability?